docs/concepts/registries.md

```diff
@@ -38,27 +38,49 @@ public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(
 );
 ```
 
-We can then add our registry entries as static final fields (see [the article on Blocks][block] for what parameters to add in `new Block()`):
+We can then add our registry entries as static final fields using one of the following methods (see [the article on Blocks][block] for what parameters to add in `new Block()`):
 
 ```java
-public static final DeferredHolder<Block, Block> EXAMPLE_BLOCK = BLOCKS.register(
-        "example_block" // Our registry name.
-        () -> new Block(...) // A supplier of the object we want to register.
+public static final DeferredHolder<Block, Block> EXAMPLE_BLOCK_1 = BLOCKS.register(
+        // Our registry name.
+        "example_block" 
+        // A supplier of the object we want to register.
+        () -> new Block(...)
+);
+
+public static final DeferredHolder<Block, SlabBlock> EXAMPLE_BLOCK_2 = BLOCKS.register(
+        // Our registry name.
+        "example_block" 
+        // A function creating the object we want to register
+        //   given its registry name as a ResourceLocation.
```

Comment:
Extra spacing in front of this line?

Reply:
I do this whenever I'm wrapping a sentence to indicate that it starts from the previous line.